electrumsv/gui/qt/account_restoration_dialog.py

```diff
@@ -478,6 +478,8 @@ def _on_clicked_button_action(self) -> None:
                 if import_entry.is_missing:
                     # We have no idea if this is in a block. We're going to try and get the proof
                     # because we have no idea if this is in a block. The wallet can sort it out.
+                    # i.e. MissingTransactionMetadata.with_proof is True but a proof may not be
```

Comment:
I do not understand what this additional comment means. It seems like a non-sequitur

Reply:
Will remove it then. It referring to how `_obtain_transactions_worker_async` attempts to fetch the merkle proof if `MissingTransactionMetadata.with_proof` is `true` but it can handle it if a proof is unavailable...